sumologic/resource_sumologic_monitors_library_monitor_test.go

```diff
@@ -800,3 +877,81 @@ func exampleMetricsMissingDataTriggerCondition(triggerType string) TriggerCondit
 		DetectionMethod: "MetricsMissingDataCondition",
 	}
 }
```

Comment:
It would be nice to add: "_driftingCorrectionFGP" testing, 
as code under `resource_sumologiic_monitors_library_monitor.go` can be accidentally modify to break drifting correction. 

If decided to add this later, we may file a small JIRA ticket to track it.

Reply:
added